lib/schemas/get_manifest.json

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
combining manifest files is a bit tricky. Trying to follow some of the work here https://github.com/rails-api/active_model_serializers/pull/1270/files and in various SO threads but so far no luck. Thoughts on merging now and fixing later? 

It is something like  "$ref": "file://lib/schemas/manifest_content.json#" but so far I can't get anything to work...